### ISSUE TITLE
Training my own classifier，after 80 epoch ，the validation is 0.85

### ISSUE THREAD

**Issue by Roysky** [davidsandberg/facenet#268]
Hello David
I have created my  own CASIA datasets which are aligned 160*160, just trying to train my own model using this datasets and  after 80 epoches , the accuracy and valication is below
76000	0.97950	0.86900
77000	0.98050	0.88367
78000	0.98083	0.89867
79000	0.97917	0.85600
80000	0.97983	0.89333

and  my training arguments are 
python src/facenet_train_classifier.py --logs_base_dir ./logs/facenet/ --models_base_dir ./models/facenet/ --data_dir /root/data/CASIA-maxpy-align160/  --image_size 160 --model_def models.inception_resnet_v1 --lfw_dir ./datasets/lfw/lfw_mtcnnpy_160 --optimizer RMSPROP --learning_rate -1 --max_nrof_epochs 80 --keep_probability 0.8 --random_crop --random_flip --learning_rate_schedule_file data/learning_rate_schedule_classifier_casia.txt --weight_decay 5e-5 --center_loss_factor 1e-4 --center_loss_alfa 0.9 --gpu_memory_fraction 0.9  

I think after 80 epoch the validation should be 0.98~0.99 , Is there something wrong with my operetion. 
Thank you

**Comment by davidsandberg** [comment]
Hi,
It looks like you are doing the training correctly. Also, the performance difference is very minor. Which dataset did you use?
The pretrained model I supplied has been trained on a [cleaned](https://groups.google.com/forum/#!topic/cmu-openface/Xue_D4_mxDQ) version of CASIA WebFace. Maybe that is the difference.

**Comment by Roysky** [comment]
yes， I don't clean the CASIA datasets.   I am not sure whether the CASIA datasets is cleaned.
how to clean the CASIA datasets or  where can I down the cleaned CASIA datasets?
thank you !

**Comment by davidsandberg** [comment]
Did you check the thread I liked to? There you can find links to the cleaned dataset.

**Comment by Roysky** [comment]
yes， I am  downing the cleaned datasets

**Comment by davidsandberg** [comment]
Closing this for now. Reopen if needed.
